dbt/models/eia860/_core_eia860__fgd_equipment/schema.yml

```diff
@@ -4,6 +4,22 @@ sources:
     tables:
       - name: _core_eia860__fgd_equipment
         data_tests:
+          - expect_columns_not_all_null:
+              exclude_columns:
+                - sorbent_type_3
+                - sorbent_type_4
+                - so2_equipment_type_4
```

Comment:
These appear to be placeholders that allow a larger number of sorbent / equipment types to be specified than anybody is actually using right now.